src/components/config-components/ConfigSelectionLayout.vue

```diff
@@ -67,6 +68,7 @@
     import ConfigSort from '../../r2mm/configs/ConfigSort';
     import FsProvider from '../../providers/generic/file/FsProvider';
     import ManagerInformation from '../../_managerinf/ManagerInformation';
+    import LinkProvider from 'src/providers/components/LinkProvider';
```

Comment:
Had a quick look at it all looks good (just need to test manually).
Could this line be relative like the other imports?

Reply:
Sure, is changed :)